OMCompiler/Compiler/BackEnd/RemoveSimpleEquations.mo

```diff
@@ -4608,6 +4608,8 @@ algorithm
       HashTableCrToExp.HashTable HTCrToExp;
       HashTableCrToCrEqLst.HashTable HTCrToCrEqLst, HTAliasLst;
       list<tuple<DAE.ComponentRef,DAE.Exp>> tplExp;
+      DAE.ComponentRef cr_print;
+      DAE.Exp exp_print;
```

Comment:
Unused, leftover of debug?

Reply:
yes, that whole thing is still pretty ugly half-converted code